### ISSUE TITLE
doctype checking too strict

### ISSUE THREAD

**Issue by nosilver4u** [keycdn/cache-enabler#250]
Just ran into an issue with the is_cacheable() method and a slightly non-standard doctype:
`<!doctype html >`
The existing check doesn't pass because of the space between html and the closing bracket, this regex fixes it:
`/^<!DOCTYPE.+html\s*>/i`
The existing regex is at https://github.com/keycdn/cache-enabler/blob/6ce1426fb53ae942f7c4cd8e96a74da6413e5fba/inc/cache_enabler_engine.class.php#L234

**Comment by coreykn** [comment]
You're right. This behavior was initially added in PR #137. Even though it is not a standard doctype, updating that regular expression to allow 0 or more whitespace characters in that location is okay by me. If you'd like, you can open a PR to close this issue. Otherwise, I can get something added at the beginning of next week.
